### ISSUE TITLE
Please update your docs

### ISSUE THREAD

**Issue by Jscott388** [mdbootstrap/Angular-Bootstrap-with-Material-Design#16]
The docs are so outdated. Nothing works properly. The <links> <logo> from the website example don't work for the navbar. This is a nightmare.  When you do NPM install there is no IMG, FONT files. Your NPM package needs updated with all the correct files needed to use from the node_module directory. Why you have to bring your scss files and typscripts files over beats me. Then you have to download the example file to get the rest of the files over for the IMG and FONT directories. Then it's a puzzle to find out to do a simple nav bar because your examples WON'T WORK!

So I was 2 seconds away from purchasing, and now I refuse to because of this.

**Comment by danielfs1** [comment]
Have to agree. I been trying to switch over to using mdbootstrap for angular the last week or two and most of the examples from the angular mdbootstrap website will not work. In fact I can download this repository, copy some UI markup from the demo website, and it won't work.

Please please please update your documentation.

**Comment by mdbootstrap** [comment]
Dear @Jscott388 @danielfs1 , we've been working on that today. We will update it tomorrow. Apologies for your inconvenience.

**Comment by Jscott388** [comment]
Can you fix the npm directory structure so we don't have to use the boilerplate project? Have all the resources there ready to go. No need to have it our actual project app folder.

**Comment by cHullaert** [comment]
Same problem, succeed to create a new project with angular-cli, think it's ok for dependencies but components don't work.  Come back tomorrow to discover the new help.  Thanks for your project, thanks for your support.

**Comment by AdrianSaw** [comment]
I guess everything should work fine now. Free npm contains everything which is needed.

**Comment by danielfs1** [comment]
Still waiting for the update that was supposed to come last week (per the statement above).

**Comment by AdrianSaw** [comment]
Could you explain more specifically what does not work?

**Comment by danielfs1** [comment]
On Sept 13th, mdbootstrap said 

> Dear @Jscott388 @danielfs1 , we've been working on that today. We will update it tomorrow. Apologies for your inconvenience.

I have not seen a update to this repo or the angular-bootstrap-md (4.3.5) NPM repo since that message.

If the only change was updating the docs then I will post my on going issues (which are still happening with the current documentation).

**Comment by ajaxon** [comment]
I'm having the same issue. I can't even copy and paste the examples on the site and have them work.

**Comment by mdbootstrap** [comment]
Dear @danielfs1 @ajaxon , could you please send us your sample projects at office@mdbootstrap.com, we will check what's going on. Docs were updated and I just tested them, they seem to be fine.

**Comment by Jscott388** [comment]
Ok. For starters. 

`<mdb-navbar SideClass="navbar navbar-expand-lg navbar-dark indigo">
           
    
    <logo><a class="navbar-brand" href="#">Navbar</a></logo>

    
    <links>

        
        <ul class="navbar-nav mr-auto">
            <li class="nav-item active">
                <a class="nav-link waves-light" mdbRippleRadius>Home<span class="sr-only">(current)</span></a>
            </li>
            <li class="nav-item">
                <a class="nav-link waves-light" mdbRippleRadius>Features</a>
            </li>
            <li class="nav-item">
                <a class="nav-link waves-light" mdbRippleRadius>Pricing</a>
            </li>

            
            <li class="nav-item dropdown" dropdown>
                <a dropdownToggle mdbRippleRadius type="button" class="nav-link dropdown-toggle waves-light" mdbRippleRadius>
                Basic dropdown<span class="caret"></span></a>
                <div *dropdownMenu class="dropdown-menu dropdown dropdown-primary" role="menu">
                    <a class="dropdown-item waves-light" mdbRippleRadius href="#">Action</a>
                    <a class="dropdown-item waves-light" mdbRippleRadius href="#">Another action</a>
                    <a class="dropdown-item waves-light" mdbRippleRadius href="#">Something else here</a>
                    <div class="divider dropdown-divider"></div>
                    <a class="dropdown-item waves-light" mdbRippleRadius href="#">Separated link</a>
                </div>
            </li>

        </ul>
        

        
        <form class="form-inline  waves-light" mdbRippleRadius>
            <input class="form-control mr-sm-2" type="text" placeholder="Search">
        </form>
    </links>
    

</mdb-navbar>`

Uncaught Error: Template parse errors:
'logo' is not a known element:

Uncaught Error: Template parse errors:
'links' is not a known element:

Copied and pasted from:
https://mdbootstrap.com/angular/components/navbars/

Remove logo and links, then navbar still won't appear. I know the resources needed such as scss files are loaded, because I can use the jquery/html version and it will render. 

<mdb-navbar> doesn't render anything. 

The angular version needs to be redone completely. All you need to do is make the directives to replace the need of jQuery. So in theory, your jQuery version and Angular HTML code should be the same. The directives should handle the stuff jQuery handled. 

There is no reason to have a <mdb-navbar> directive.  There isn't any jQuery function of it. The css / html code should do the work. 

Things like:
Accordion
Alert
Buttons
Carousel
Collapse
Datepicker
Dropdown
Modal
Pagination
Popover
Progressbar
Rating
Tabs
Timepicker
Tooltip

These would need the directives. In fact since this is bootstrap, you could get these directives from ng-bootstrap to start off with. The based project is done, you just need to add your PRO version directives. I'd be happy to help in anyway I can for in exchange of the pro version!!

**Comment by wtkaczyk** [comment]
Nice try Jscott388 :) 

Probably it's caused missing <mdb-navbar> statement in your example.

**Comment by Jscott388** [comment]
I think I’m going abandoned this project. The support here is lacking and I don’t know what docs you updated but copy and paste the examples does not work. So charge for the pro version when none of the examples work is beyond comprehension.

**Comment by mdbootstrap** [comment]
Dear @Jscott388 , 

Within your code snippet you have missed totally navbar tag: 

```
<!--Navbar-->
<mdb-navbar SideClass="navbar navbar-expand-lg navbar-dark indigo">
    ... 
</mdb-navbar>
<!--/.Navbar-->
            
```

Please copy entire example from documentation and check again.

**Comment by vamshi9** [comment]
@mdbootstrap [https://stackoverflow.com/questions/46930046/mdb-bootstrap-not-working-out-properly](nothing is working out )

**Comment by mdbootstrap** [comment]
@vamshi9 , first of all this is angular version, while question on stack regards jQuery version. Secondly it was already answererd - MDB4 is based on Bootstrap 4 and will not work with Bootstrap 3.

**Comment by lyudmilpetrov** [comment]
Guys very unprofessional the same issue when copying a sample
from your website

Can't bind to 'containerInside' since it isn't a known property of 'mdb-navbar'.

The pasted code that is not working is

<mdb-navbar SideClass="navbar fixed-top navbar-expand-lg navbar-dark bg-pink scrolling-navbar" [containerInside]="false">
    <logo>
        <a class="logo navbar-brand" href="#"><strong>Navbar</strong></a>
    </logo>
    <links>
        <ul class="navbar-nav mr-auto">
            <li class="nav-item active waves-light" mdbRippleRadius>
                <a class="nav-link">Home <span class="sr-only">(current)</span></a>
            </li>
            <li class="nav-item waves-light" mdbRippleRadius>
                <a class="nav-link">Features</a>
            </li>
            <li class="nav-item waves-light" mdbRippleRadius>
                <a class="nav-link">Pricing</a>
            </li>
            <li class="nav-item waves-light" mdbRippleRadius>
                <a class="nav-link">Opinions</a>
            </li>
        </ul>
        <ul class="navbar-nav nav-flex-icons">
            <li class="nav-item waves-light" mdbRippleRadius>
                <a class="nav-link"><i class="fa fa-facebook"></i></a>
            </li>
            <li class="nav-item waves-light" mdbRippleRadius>
                <a class="nav-link"><i class="fa fa-twitter"></i></a>
            </li>
            <li class="nav-item waves-light" mdbRippleRadius>
                <a class="nav-link"><i class="fa fa-instagram"></i></a>
            </li>
        </ul>
    </links>
</mdb-navbar>

<main>
    <div class="container" style="height:1300px;">
        <div class="row mt-5 pt-5">
            <div class="col text-center">
                <h2>This Navbar is fixed</h2>
                <h5>It will always stay visible on the top, even when you scroll down</h5>
                <br>
                <p>Full page intro with background image will be always displayed in full screen mode, regardless of device </p>
            </div>
        </div>
    </div>
</main>

**Comment by mdbootstrap** [comment]
@lyudmilpetrov , please format your code properly. Paste the code and use code (<> icon) from menu bar.

**Comment by jkaldon** [comment]
I had this same problem and I think I know what people are doing wrong.  It took me many hours to figure out my mistake.

I started by navigating to https://mdbootstrap.com/angular/ and downloaded mdb-angular-free-5.0.5.zip.  I unzipped the project and opened it in Visual Studio Code.  Now I wanted to try some examples.

So, I went and googled 'mdbootstrap modal examples' and naively clicked on the first link.  The problem, is that the first link is to https://mdbootstrap.com/javascript/modals/ when what I really wanted was this link: https://mdbootstrap.com/angular/advanced/modals/

Bottom line, make sure you're using examples from the section of the site that corresponds with your framework.

Thanks,
- Joshua

**Comment by lyudmilpetrov** [comment]
Thank you

**Comment by SudhirKaushik** [comment]
Not agreed with @jkaldon 

I copied the angular based snippet from https://mdbootstrap.com/angular/components/navbars/

and got this error on console: 

_Uncaught Error: Template parse errors:
'logo' is not a known element_

However, removing <logo> and <links> tag made the navbar appear. Mdbootstrap version is 5.0.5.

I can feel the frustration @Jscott388 is having.

**Comment by aleek** [comment]
+1 for outdated docs. Please update them.

**Comment by PogoMips** [comment]
@SudhirKaushik Did you include schemas: [ NO_ERRORS_SCHEMA ] in your app module? It's in the setup guide, but it's easy to overlook...

**Comment by SudhirKaushik** [comment]
I tried and got `Uncaught ReferenceError: NO_ERRORS_SCHEMA is not defined` in console.

Scores:
Ignorance - 1
Proper documentation - 0

**Comment by CodependentDigital** [comment]
Whats happening here guys? I bought pro, but expected a lot more that whats happening here. All your directives are broken..

**Comment by louisamoros** [comment]
Same issue with the release `5.1.0`. The navbar is not working because of the `links` and `logo` attributes. When I removed them or put them like `<ng-container logo></ng-container>` to inject the inner attribute inside the `mdb-navbar` component, responsive is not working, burger menu does not work. Any updates? Issue is not closed.

**Comment by mdbootstrap** [comment]
Have you included schemas: [ NO_ERRORS_SCHEMA ] in your app module?

**Comment by SudhirKaushik** [comment]
Looks like a BOT is talking to us.

**Comment by mdbootstrap** [comment]
@SudhirKaushik , the error which you provided indicate issue with missing NO_ERROR_SCHEMA. If you that doesn't solve the issue , please provide more input to your project - attach project and provide environment details (OS, Angular, CLI, TS versions).

**Comment by izverev-aexsoft** [comment]
@mdbootstrap I have same issue with the carousel (code copied from here: https://mdbootstrap.com/angular/advanced/carousel/): "Can't bind to 'isControls' since it isn't a known property of mdb-carousel"
I already had NO_ERRORS_SCHEMA in top-level module but had to add it in the actual child module to get it work.
Then error disappeared, but it still won't display properly, all I can see is just three images, no controls, no animation. 
Something else should be done and it is not documented.

**Comment by atul-shimpi** [comment]
anyone successfully able to use nav bars with dropdown ? I cant see the dropdown menu

**Comment by stephenwilson1984** [comment]
No luck here either, I'm getting the Can't bind to 'containerInside' since it isn't a known property of 'mdb-navbar'. error myself.  Scrapping mdb, will try another framework.

**Comment by gopalkrish23** [comment]
@stephenwilson1984 I installed angular-bootstrap-md and followed their instructions it worked fine.

import { NgModule, NO_ERRORS_SCHEMA } from '@angular/core';
import { MDBBootstrapModule } from 'angular-bootstrap-md';
 
@NgModule({
    imports: [
        MDBBootstrapModule.forRoot()
    ],
    schemas: [ NO_ERRORS_SCHEMA ]
});
in angular.json change:
"styleExt": "css" to "styleExt": "scss"

rename /src/styles.css to styles.scss

if you want to change styles in exisiting project you can use ng config schematics.@schematics/angular:component.styleext scss

add below lines to angular.json:

"styles": [
    "node_modules/font-awesome/scss/font-awesome.scss",
    "node_modules/angular-bootstrap-md/assets/scss/bootstrap/bootstrap.scss",
    "node_modules/angular-bootstrap-md/assets/scss/mdb.scss",
    "src/styles.scss"
],
"scripts": [
  "node_modules/chart.js/dist/Chart.js",
  "node_modules/hammerjs/hammer.min.js"
],
install external libs
npm install -–save chart.js@2.5.0 font-awesome hammerjs
